modules/backend/noPingReply.js

```diff
@@ -0,0 +1,22 @@
+
+let version = '1.0';

```

Comment:
This should probably follow semantic versioning (1.0.0 instead of 1.0), but minor and just for consistency haha